package/etc/conf.d/filters/cisco/esa.conf

```diff
@@ -0,0 +1,44 @@
+filter f_cisco_esa {
+    match("cisco_esa_12_5", value("fields.sc4s_vendor_product") type(glob))
+};
+
+filter f_cisco_esa_http {
+    match('https?\s*service\s*', value("MESSAGE"))
+    or match('[sS]ession\s*\w+\s*from\s*', value("MESSAGE"))
+    or match('\s*PERIODIC\s*REPORTS\s*:\s*', value("MESSAGE"))
+    or match('\s*req\s*:.+\s*user\s*:\s*.+\s*id\s*:\s*', value("MESSAGE"))
+    or match('testmaillog\s*:\s*[iI]nfo\s*:', value("MESSAGE"))
+    or match('[sS]ystem\s*is\s*coming\s*up', value("MESSAGE"))
+};
+
+filter f_cisco_esa_cef {
```

Comment:
Cef formats should not be custom filters we can use the common log path for that

Reply:
Okay, Removed the **f_cisco_esa_cef** filter